ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ValueLimitsIT.scala

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.Assertions._
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.test.model.Test.DummyWithAnnotation
+
+import scala.concurrent.Future
+import scala.util.Random
+
+final class ValueLimitsIT extends LedgerTestSuite {
+
+  test(
+    "VLLargeSubmittersNumberCreateContract",
+    "Create a contract with a large submitters number",
+    allocate(NoParties),
+    runConcurrently = false,
```

Comment:
Added this to avoid `Overloaded` errors on the `sandbox-on-x`. Couldn't figure out what was the reason.

Reply:
After discussing with @nmarton-da we decided to increase the `ReadWriteServiceBridge` queue size. The previous value (200) was too low to handle conformance tests load.
It'd be valuable to implement a retry mechanism in `ReadWriteServiceBridge.submit()` in the `sandbox-on-x`.